subprojects/docs/src/docs/release/notes.md

```diff
@@ -1,8 +1,22 @@
-The Gradle team is pleased to announce the release of Gradle 4.1.
+The Gradle team is pleased to announce the first release candidate for Gradle 4.1.
```

Comment:
I think we always used the removed sentence. The reason is that we will then no need to change the notes between RCs. Has there been some discussion to change this to always change the release notes between RCs and the final release?